src/BloomExe/XmlHtmlConverter.cs

```diff
@@ -300,6 +305,14 @@ public static string SaveDOMAsHtml5(XmlDocument dom, string targetPath)
 			return targetPath;
 		}
 
+		private static void RemoveUiElements(XmlDocument dom)
```

Comment:
Thanks for pushing me here... found a bug in the code that was removing bloom-ui elements so that they still got sucked into the dataDiv. Fixed that so now we don't need this RemoveUiElements method.